lib/godwoken_explorer/graphql/resolovers/transaction.ex

```diff
@@ -230,22 +230,32 @@ defmodule GodwokenExplorer.Graphql.Resolvers.Transaction do
 
   defp transactions_order_by(query, input) do
     sorter = Map.get(input, :sorter)
+    status = Map.get(input, :status)
 
-    if sorter do
-      order_params = cursor_order_sorter(sorter, :order, @sorter_fields)
-      order_by(query, [u], ^order_params)
+    if status == :pending do
+      order_by(query, [u], [:hash])
     else
-      order_by(query, [u], @default_sorter)
+      if sorter do
+        order_params = cursor_order_sorter(sorter, :order, @sorter_fields)
```

Comment:
if user call api only with :block_number, the order_params will not conform to uniq property

can change like token transfer do:
```
    if sorter do
      order_params = cursor_order_sorter(sorter, :order, @sorter_fields)
      base = Enum.map(order_params, fn {_, e} -> e end)
      extend = [:transaction_hash, :log_index] -- base
      extend = extend |> Enum.map(fn e -> {:asc, e} end)

      order_params = order_params ++ extend
      order_by(query, [u], ^order_params)
    else
      order_by(query, [u], [:block_number, :transaction_hash, :log_index])
    end
```